sonar-project.properties

```
@@ -0,0 +1,13 @@
+# must be unique in a given SonarQube instance
+sonar.projectKey=CA-MMIS:HELLO
```

Super small nitpick:

```suggestion
sonar.projectKey=CA-MMIS:HELLO_CAMMIS
```

---

Done!